api4/emoji_test.go

```diff
@@ -238,9 +238,9 @@ func TestGetEmojiList(t *testing.T) {
 			found = true
 			break
 		}
-		if found {
-			t.Fatalf("should not get a deleted emoji %v", emojis[0].Id)
-		}
+	}
+	if found {
+		t.Fatalf("should not get a deleted emoji %v", emojis[0].Id)
 	}
```

Comment:
I recommend to change this to ``require.False(t, found, "Should not get a deleted emoji")``

Reply:
Sure, no worries. I just wanted to keep the changes focused only on the ineffective assignments. Will update.